content/en/docs/concepts/storage/volumes.md

```diff
@@ -118,17 +115,18 @@ are redirected to the `disk.csi.azure.com` {{< glossary_tooltip text="CSI" term_
 The AzureDisk in-tree storage driver was deprecated in the Kubernetes v1.19 release
 and then removed entirely in the v1.27 release.
 
-The Kubernetes project suggests that you use the [Azure Disk](https://github.com/kubernetes-sigs/azuredisk-csi-driver) third party
-storage driver instead.
+The Kubernetes project suggests that you use the [Azure Disk](https://github.com/kubernetes-sigs/azuredisk-csi-driver)
+third party storage driver instead.
 
 ### azureFile (deprecated) {#azurefile}
 
 {{< feature-state for_k8s_version="v1.21" state="deprecated" >}}
 
 The `azureFile` volume type mounts a Microsoft Azure File volume (SMB 2.1 and 3.0)
-into a pod.
+in a pod.
```

Comment:
I would keep the original text here.

You mount things "into" or "onto" or "on to". It's by analogy with the physical meaning of "mount".

For example, you can mount a painting on to a wall. You don't mount a painting in a wall.